package.json

```diff
@@ -63,8 +63,8 @@
           "items": {
             "type": "string"
           },
-          "default": [],
-          "description": "List of command line options that gets passed in to javac. See available options in: https://checkerframework.org/manual/#running"
+          "default": ["-AlspTypeInfo"],
```

Comment:
```suggestion
          "default": ["-proc:only", "-AlspTypeInfo"],
```

Reply:
This is an array of strings, right? Can you test whether this works as expected?

Reply:
Yes, and yes it does work as expected in my tests.